hack/genpkgs/load.sh

```diff
@@ -6,8 +6,11 @@ __dir=$(cd "$(dirname "$0")" && pwd)
 
 go run ${__dir}/main.go > ${__dir}/data.csv
 
-mysql -h127.0.0.1 -uroot -proot123 -P7450 enduro \
+mysql -h127.0.0.1 -uroot -proot123 -P3306 enduro \
+	-e "SET GLOBAL local_infile=1;"
```

Comment:
Really unlikely to be used outside of a development/test environment, but I'd change the global back at the end of the script, just in case.